types/office-js/index.d.ts

```diff
@@ -17176,9 +17176,12 @@ declare namespace OfficeExtension {
 ////////////////////////////////////////////////////////////////
 
 
```

Comment:
Adding spacing here so it is consistent with spacing in the rest of the file, I think.